src/main/java/org/folio/circulation/resources/SlipsResource.java

```diff
@@ -118,6 +120,22 @@ private void getMany(RoutingContext routingContext) {
       return;
     }
 
+    configurationRepository.lookupPrintHoldRequestsEnabled().thenAccept(result -> {
+        if (result.succeeded()) {
+          PrintHoldRequestsConfiguration printHoldRequestsConfiguration = result.value();
+          if (printHoldRequestsConfiguration == null ||
+            !printHoldRequestsConfiguration.isPrintHoldRequestsEnabled()) {
+            context.writeResultToHttpResponse(succeeded(JsonHttpResponse.ok(new JsonObject())));
```

Comment:
You need to interrupt the execution at this point, otherwise the rest of the code in this method will be executed as well